Principal_component_analysis/include/CGAL/linear_least_squares_fitting_segments_2.h

```diff
@@ -104,9 +104,9 @@ linear_least_squares_fitting_2(InputIterator first,
 
   // Translate the 2nd order moment calculated about the origin to
   // the center of mass to get the covariance.
-  covariance[0] += -mass * ( c.x() * c.x());
-  covariance[1] += -mass * (c.x() * c.y());
-  covariance[2] += -mass * (c.y() * c.y());
+  covariance[0] -= mass * CGAL::square(c.x()));
```

Comment:
One parenthesis too much.    As soon as a PR is marked for Under Testing this means that it got integrated in the testsuite.   There you can see that the line of Principal Component Analysis is fully red.     It is best practice to compile and run the trstsuite of a package that gets modifed locally before pushing.

Reply:
My bad. Will follow that procedure.